spec.bs

```diff
@@ -639,6 +641,21 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
     [=auction config/per buyer experiment group ids=].
   1. If |perBuyerExperimentGroupIds| is not null and |perBuyerExperimentGroupIds|[|buyer|] exists:
     1. Set |buyerExperimentGroupId| to |perBuyerExperimentGroupIds|[|buyer|].
+  1. Let |buyerGroupLimit| be |allBuyersGroupLimit|.
+  1. Let |perBuyerGroupLimits| be |auctionConfig|'s
+    [=auction config/per buyer group limits=].
+  1. If |perBuyerGroupLimits| is not null and |perBuyerGroupLimits|[|buyer|] exists:
+    1. Set |buyerGroupLimit| to |perBuyerGroupLimits|[|buyer|].
+  1. Let |igs| be a new [=list=] of [=interest groups=].
+  1. [=map/For each=] signalsUrl -> |perSignalsUrlGenerator| of |perBuyerGenerator|:

```

Comment:
For consistency with the rest of the spec I'd probably try and use this little arrow thingy: → here and below in this PR.